### PR TITLE
improvement: add more types to the openapi schema

### PR DESCRIPTION
**Pull request by mscolnick**
## 📝 Summary

Add more open-api generated datatypes

### CODE REVIEW

**Comment by mscolnick** [comment]
will merge after #1580  to avoid the merge conflict

File: marimo/_cli/development/commands.py
Comment (akshayka) [inline marimo/_cli/development/commands.py:35]: Aren't these CamelCase?

**Review by akshayka — DISMISSED** [review]

File: marimo/_cli/development/commands.py
Comment (mscolnick) [reply marimo/_cli/development/commands.py:35]: they are not, they are snake case. and not sure we can migrate to cameCase since the `data` attribute on CellOutput would be converted too. 

i think we could if we make `deep_camel_case` only operate on dataclass fields

Comment (akshayka) [reply marimo/_cli/development/commands.py:34]: Oh, I was just looking at the class names, but the field names are snake case, which is what you must be referring too. Got it, makes sense

**Pull request merged**
